static/views/updater.html

```diff
@@ -25,7 +25,7 @@
         button {
             cursor: pointer;
             padding: 0.5em;
-            color: var(--fg);
+            color: var(--bg);
```

Comment:
huh?

Reply:
This looks better on some themes and doesn't on others. Unfortunately, there isn't really a way to find the actual Discord button styles without hardcoding class names and injecting fake HTML, so maybe keep the buttons as-is?